scraper/src/documentation_spider.py

```diff
@@ -24,6 +24,8 @@ class DocumentationSpider(CrawlSpider, SitemapSpider):
     """
     DocumentationSpider
     """
+    http_user=os.environ.get('DOCSEARCH_BASICAUTH_USERNAME', None)
```

Comment:
What is the use of these vars?

Reply:
These vars will provide HTTP basic auth username and password. 
I think that these vars should be provided in config file, but these are passwords, so better read the information from environment variables.

Reply:
Sorry, I think that I'm misunderstanding your question. The "http_user" is provided by scrapy.